external/fv3fit/fv3fit/pytorch/GraphHyperparameters.yml

```diff
@@ -0,0 +1,23 @@
+- name: GraphHyperparameters
```

Comment:
Why is there a configuration file added to the package? Ideally all configuration should be user-provided.